tests/test_parameters/test_parameter_values.py

```diff
@@ -12,15 +12,15 @@ def test_read_parameters_csv(self):
         data = pybamm.ParameterValues().read_parameters_csv(
             "input/parameters/lead-acid/default.csv"
         )
-        self.assertEqual(data["Ideal gas constant"], 8.314)
+        self.assertEqual(data["Reference temperature"], 294.85)
```

Comment:
Can we have a test: approxEqual(R, 8.314), and approxEqual(F, 96487) just incase something weird was to happen and one of these values changed in scipy?